catalog-info.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: terraform-gcp-gke
+  description: Terraform module for creating Google Kubernetes Engine in Google Cloud Platform
+  annotations:
+    github.com/project-slug: honestbank/terraform-gcp-gke
+    github.com/team-slug: honestbank/devops-engineers
+spec:
+  type: tf-module
```

Comment:
@deejiw have we standardized these types somewhere? Appreciate if you can point me to the resource thanks. Not sure why this isn't classed the same as core-api-lib? Cc @MXfive @poom

Reply:
Referred link https://www.notion.so/honestbank/Using-Backstage-Service-Catalog-111c46b302174d0e8f6eaa4859e5e967
For Resource, there are `database`, `storage`, `tf-module` and `tf-workspace`.